src/main/java/org/broadinstitute/hellbender/cmdline/argumentcollections/RequiredReadInputArgumentCollection.java

```diff
@@ -1,19 +0,0 @@
-package org.broadinstitute.hellbender.cmdline.argumentcollections;
-
-import org.broadinstitute.hellbender.cmdline.Argument;
-import org.broadinstitute.hellbender.cmdline.ArgumentCollectionDefinition;
-import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
-
-import java.io.File;
-import java.util.List;
-
-/**
- * An argument collection for use with tools that accept one or more input files containing reads
- * (eg., BAM/SAM/CRAM files), and require at least one such input.
- */
-public class RequiredReadInputArgumentCollection implements ArgumentCollectionDefinition {
-
-    @Argument(fullName = StandardArgumentDefinitions.INPUT_LONG_NAME, shortName = StandardArgumentDefinitions.INPUT_SHORT_NAME, doc = "One or more BAM/SAM/CRAM files containing reads", optional = false)
-    public List<File> readFiles;
```

Comment:
These argument collections are part of the solution to the required/optional input issue addressed in https://github.com/broadinstitute/hellbender/pull/440/files and should not be deleted.